src/GDGUkraine/lib/utils/table_exporter.py

```diff
@@ -1,8 +1,41 @@
 from io import BytesIO
+from json import dumps as json_dumps
+
 from openpyxl import Workbook
 from openpyxl.writer.excel import save_virtual_workbook
 
 
+event_parsers = {
```

Comment:
Not sure if it is a good idea to have concrete report specific logic here, but while we have only one report, it is plausible.